tools/common.py

```diff
@@ -0,0 +1,10 @@
+def includeDirOrder():
+  return (
+      "envoy",
+      "common",
+      "source",
```

Comment:
Should we just block including using "source/"? @htuch do you recall how this ended up here? I don't remember now.

Reply:
recursive grep tells me there are exactly two files in the repo that include "source/..":

./source/common/ratelimit/ratelimit_impl.h:#include "source/common/ratelimit/ratelimit.pb.h"
./test/integration/ratelimit_integration_test.cc:#include "source/common/ratelimit/ratelimit.pb.h"

Reply:
Also I want to suggest not having a policy-change like this be a new responsibility of this PR...happy to hack that as a follow up if you like :)

Reply:
Sure makes sense. I was just curious.